import_export/mixins.py

```diff
@@ -86,3 +116,6 @@ def form_valid(self, form):
 
         post_export.send(sender=None, model=self.model)
         return response
+
+    def get_export_filename(self, file_format):
```

Comment:
I did this because the two calls to `get_export_filename()` use different args. 

- [admin.py](https://github.com/django-import-export/django-import-export/blob/722c376e476f0a9b5252c21d4cf788c9bdd1bdd8/import_export/admin.py#L483) calls `get_export_filename(request, queryset, file_format)`
- [mixins.py](https://github.com/django-import-export/django-import-export/blob/722c376e476f0a9b5252c21d4cf788c9bdd1bdd8/import_export/mixins.py#L84) calls `get_export_filename(file_format)`

I wanted to avoid changing existing logic, in case it has a side-effect which I didn't know about, so I used other methods to wrap the existing calls.

Reply:
right! didn't notice that, thanks!